src/sbvr-api/uri-parser.ts

```diff
@@ -89,7 +89,7 @@ export const memoizedParseOdata = (() => {
 		max: env.cache.parseOData.max,
 	});
 	return (url: string) => {
-		if (_.includes(url, '$')) {
+		if (_.includes(url, '?')) {
```

Comment:
Don't we also want to skip caching plain requests such as `/v5/device(12345)`?

Reply:
Potentially but I think this covers the most important cases (based on checking cache entries in the api).  I also wonder whether with the much faster odata-parser if it's worth having this cache at all but that's a separate discussion